src/client/Microsoft.Identity.Client/ManagedIdentity/Constants.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    internal class Constants
+    {
+        public const string OrganizationsTenantId = "organizations";
```

Comment:
Just copy pasted this class. Will clean it up once all the MI sources are integrated.